run-build-functions.sh

```diff
@@ -779,6 +817,56 @@ restore_cwd_cache() {
   move_cache "$NETLIFY_CACHE_DIR/$1" "$PWD/$1" "restoring cached $2"
 }
 
+#
+# Restores node_modules dirs cached for js workspaces
+# See https://github.com/netlify/pod-workflow/issues/139/ for more context
+#
+# Expects:
+# $@ each argument should be a package location relative to the repo's root
+restore_js_workspaces_cache() {
+  local locations=("$@")
+  local cache_dir="$NETLIFY_CACHE_DIR/js-workspaces"
```

Comment:
Should this be moved to the top-level scope? To avoid it being declared twice.

Reply:
Yeah I see what you mean, I ended up going with that pattern as it was something already being done for the other cache directories. But makes sense IMO. Addressed via https://github.com/netlify/build-image/commit/14b1ea8cb00c87c1b649b344b5ebd6f94683aedc.